app/models/planning_application.rb

```diff
@@ -217,9 +217,9 @@ def full_address
 
   def secure_change_url(application_id, secure_token)
     if ENV["DOMAIN"] == "bops-staging.services"
```

Comment:
Am I missing something here? Why do we have this `if` statement?

Could this method just be the last line? `"http://#{local_authority.subdomain}.#{ENV['APPLICANTS_APP_HOST']}/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"`

Reply:
I didn't add the if statement, all I did to this was change everything to http - it was introduced by @Rhian on the email PR - Rhian do you recall the reason for it?

Reply:
It was a workaround yesterday because there was an issue with environment variables on Preview that I couldn't resolve quickly